tests/test_serializable_collection.py

```diff
@@ -71,6 +71,54 @@ def test_repr(self):
             ")"
         )
 
+    def test_children_if(self):
+        cl = otio.schema.Clip()
+        tr = otio.schema.Track()
+        tr.append(cl)
+        tl = otio.schema.Timeline()
+        tl.tracks.append(tr)
+        sc = otio.schema.SerializableCollection()
+        sc.append(tl)
+        result = sc.children_if(otio.schema.Clip)
+        self.assertEqual(len(result), 1)
+        self.assertTrue(result[0], cl)
+
+    def test_children_if_search_range(self):
+        range = otio.opentime.TimeRange(
+            otio.opentime.RationalTime(0.0, 24.0),
+            otio.opentime.RationalTime(24.0, 24.0))
+        cl0 = otio.schema.Clip()
+        cl0.source_range = range
+        cl1 = otio.schema.Clip()
+        cl1.source_range = range
+        cl2 = otio.schema.Clip()
+        cl2.source_range = range
+        tr = otio.schema.Track()
+        tr.append(cl0)
+        tr.append(cl1)
+        tr.append(cl2)
+        tl = otio.schema.Timeline()
+        tl.tracks.append(tr)
+        sc = otio.schema.SerializableCollection()
+        sc.append(tl)
+        result = sc.children_if(otio.schema.Clip, range)
+        self.assertEqual(len(result), 1)
+        self.assertTrue(result[0], cl0)
```

Comment:
Is this also meant to be `assertEqual`?
```suggestion
        self.assertEqual(result[0], cl0)
```

Reply:
Oops, sorry, I forgot the Python tests. Should be fixed now.